zk.js/tests/testData/tmp_test_psp.ts

```diff
@@ -333,7 +333,7 @@ export type TmpTestPsp = {
       };
     },
     {
-      name: "utxo";
+      name: "utxoOutUtxo";
```

Comment:
why utxo twice in the name?

Reply:
Because the idl generates a type and an object with the same fields.